lec04-dplyr.Rmd

```diff
@@ -645,7 +635,7 @@ surveys_abun_species %>%
 
 It is possible to specify exactly which colors to use, to avoid those that are
 hard to distinguish. We can also change the thickness of the lines, and adjust
-the x labels so that they don't overlap. This will be discussed in detail in the
+the x labels so that they don't overlap. 
 data visualization lecture.
```

Comment:
Did you not also want to remove the end of the sentence?

Reply:
...whoops

Good catch!